odoo/addons/base/models/res_users.py

```diff
@@ -1915,6 +1926,17 @@ def change_password(self):
         # would be great to update the session id in-place, but it seems dicey
         return {'type': 'ir.actions.client', 'tag': 'reload'}
 
+class RevokeAllDevices(models.TransientModel):
+    _inherit = 'res.users.identitycheck'
```

Comment:
the `_inherit` is if you are modifying the model in another module. As you are in base, you can modify the main model definition directly.